### ISSUE TITLE
Get merge layer's output

### ISSUE THREAD

**Issue by rogerchen1226** [fchollet/keras#4351]
Hi, please help me~~
I want to get the output of the final_model which merged two model, here is an example.

`from keras.layers import Merge`
`left_branch = Sequential()`
`left_branch.add(Dense(32, input_dim=784))`
`right_branch = Sequential()`
`right_branch.add(Dense(32, input_dim=784))`
`merged = Merge([left_branch, right_branch], mode='concat')`
`final_model = Sequential()`
`final_model.add(merged)`
`final_model.add(Dense(100, activation='relu'))`
`final_model.add(Dropout(0.5))`
`final_model.add(Dense(10, activation='softmax'))`
`final_model.fit([left.X,right.X],train_y)`

I try the document and only work on left_branch or right_branch by single input.
And I try to input the `[left.X,right.X]` in `get_output ` function didn't work.
How to use the K.function in that situation. Please give me some suggestions, thank you. 

`from keras import backend as K`
`get_output= K.function([final_model.layers[1].input,K.learning_phase()],[final_model.layers[1].output,0])`
`layer_output = get_output([[left.X,right.X],0])[0]`

**Comment by kilotaras** [comment]
Can you use `model.predict`?

**Comment by longma307** [comment]
https://github.com/fchollet/keras/issues/2310
When I was trying to get the output, I still have the error as the above thread discussed.
I tried their method, no luck to fix that.

**Comment by rogerchen1226** [comment]
@kilotaras I want to get the `final_model.add(Dense(100, activation='relu'))` this layer, not the softmax layers

**Comment by rogerchen1226** [comment]
@kilotaras i got it!!! very very thank you ^^

**Comment by litingsjj** [comment]
@rogerchen1226 @kilotaras   I have a same problem! And I still don't know how to run get_output() correct. What's meaning of create another model? Because I want use the features in SVM. Can you tell me how to fix it? thank you!
